components/locid/src/langid.rs

```diff
@@ -85,6 +88,22 @@ impl LanguageIdentifier {
         parse_language_identifier(v, ParserMode::LanguageIdentifier)
     }
 
+    #[doc(hidden)]
+    // We cannot return `Result<Self, ParserError>`, as it's currently impossible to unwrap non-Copy
+    // types in a const context. See [rust-lang#73255](https://github.com/rust-lang/rust/issues/73255).
```

Comment:
`suggestion`: the comment would be more actionable if it was phrased as "Switch to `Result<Self, ...>` once X stabilizes. See ...`